diff --git a/node_modules/moment/moment.js b/node_modules/moment/moment.js
index 6b922fc..beed526 100644
--- a/node_modules/moment/moment.js
+++ b/node_modules/moment/moment.js
@@ -619,7 +619,7 @@
     function unescapeFormat(s) {
         return regexEscape(
             s
-                .replace('\\', '')
+                .replace(/\\/g, '')
                 .replace(
                     /\\(\[)|\\(\])|\[([^\]\[]*)\]|\\(.)/g,
                     function (matched, p1, p2, p3, p4) {
